lib/main_common.pm

```diff
@@ -1438,7 +1438,7 @@ sub load_extra_tests_desktop {
     # well
     if (check_var('DESKTOP', 'gnome')) {
         loadtest 'x11/yast2_lan_restart';
-        loadtest 'x11/yast2_lan_restart_devices';
+        loadtest 'x11/yast2_lan_restart_devices' unless (is_opensuse && !is_leap('<=15.0'));
```

Comment:
Hmm, seems that I miss something, but haven't you adjusted this test suite to work with network manager?
Also, multiple conditions with `unless` are not as readable as it's `if` version.

Reply:
This test was a time ago a big test and I split this scenario in two. In yast2_lan_restart we only check tabs as most of the thing are disabled. yast2_lan_restart_devices is testing some 'more advance' configuration, so the whole test does't make sense to schedule it.
Let me fix the condition, think is not so cumbersome, my mistake.

Reply:
Ah, I see. Good approach, thanks for the clarification!